.travis.yml

```diff
@@ -9,7 +9,9 @@ services:
 before_install:
   - sudo apt-get -qq update
 install:
-  - pip install molecule
+  - pip install molecule==2.22rc1 
```

Comment:
Please use a stable version of molecule instead of a release candidate.

Reply:
unfort. `2.22` not released yet (details in my comment below)

Reply:
fixed now with released `v2.22` (see below)